src/style.css

```
@@ -0,0 +1,26 @@
+html {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+  
+}
```

```suggestion
h1 {
  font-size: 17px;
  font-weight: bold;
}
```
Redundant empty line